CHANGELOG.md

```diff
@@ -276,6 +276,7 @@ All notable changes to this project will be documented in this file.
 [`while_let_on_iterator`]: https://github.com/Manishearth/rust-clippy/wiki#while_let_on_iterator
 [`wrong_pub_self_convention`]: https://github.com/Manishearth/rust-clippy/wiki#wrong_pub_self_convention
 [`wrong_self_convention`]: https://github.com/Manishearth/rust-clippy/wiki#wrong_self_convention
+[`wrong_transmute`]: https://github.com/Manishearth/rust-clippy/wiki#wrong_transmute
```

Comment:
Can you create a *future version* section if necessary and add the link in a *New lint{,s}* item?